### PR TITLE
Tabs feature. Set HTML attributes to `tab-content`  container

### PR DESCRIPTION
**Pull request by AndrewKorpusov**
| Q | A |
| --- | --- |
| Is bugfix? | no |
| New feature? | yes |
| Breaks BC? | yes |
| Tests pass? | yes |
| Fixed issues | - |

### CODE REVIEW

File: Tabs.php [hunk 1]
```diff
@@ -115,6 +115,10 @@ class Tabs extends Widget
      */
     public $renderTabContent = true;
 
+    /**
+     * @var array list of Html attributes for the `tab-content` container
+     */
```
Comment (cebe) [inline Tabs.php]: needs `@since 2.0.7` annotation.

File: Tabs.php [hunk 2]
```diff
@@ -123,6 +127,8 @@ public function init()
     {
         parent::init();
         Html::addCssClass($this->options, ['widget' => 'nav', $this->navType]);
+        if ($this->renderTabContent)
```
Comment (cebe) [inline Tabs.php]: no need for the condition imo.

**Comment by cebe** [comment]
also need a changelog entry.

**Comment by SilverFire** [comment]
Could you also update [tests](https://github.com/yiisoft/yii2-bootstrap/blob/master/tests/TabsTest.php), please?

**Comment by cebe** [comment]
Merged, thank you!

**Pull request closed without merging**